src/lib.rs

```diff
@@ -5,20 +5,14 @@
 //! It allows to simulate chained trades over different venues
 //! together to exploit price differences by using token prices
 //! calculated from the protocol's state.
-//!
-//! The main data structure is a graph which allows to search
-//! for a sequence of swaps that provide some desired outcome,
-//! such as atomic arbitrage opportunities. The graph models
-//! each token as a node and decentralised exchange protocols
-//! as edges.
-//!
-//! The crate also provides optimization methods, such as golden
-//! section search, to find optimal amounts for a specific sequence.
 
 extern crate core;
 
 // Reexports
-pub use num_traits;
+pub use tycho_client;
+pub use tycho_core;
+pub use tycho_ethereum;
```

Comment:
Is this one required to be reexported?

Reply:
tycho_ethereum? huumm I'm not sure. If I'm being honest I don't even know what's in it 😅

Reply:
it's actually not even used in the simulation repo.. 🤷🏼‍♀️  removing it